docs/changelog.rst

```diff
@@ -43,6 +43,15 @@ New features:
   disambiguation comment. A new ``releasegroupdisambig`` field has been added.
   :bug:`3024`
 
+Changes:
+
+* :doc:`/plugins/mbsync` no longer queries MusicBrainz when the either the
+  ``mb_albumid`` or ``mb_trackid`` field is invalid
+  Discussion on Google groups here_
```

Comment:
To avoid “using up” the link word “here” for all time, how about this more descriptive link text?

> See also the [discussion on Google Groups].

where [] indicated where the link should go.

Reply:
I think I got this one right